model/plotWidget.h

```diff
@@ -86,7 +86,9 @@ namespace minsky
     float height() const {return Item::height();}
     
     const PlotWidget* plotWidgetCast() const override {return this;}
-    PlotWidget* plotWidgetCast() override {return this;}        
+    PlotWidget* plotWidgetCast() override {return this;}
+    
+    std::string classType() const override {return "PlotWidget";}            
```

Comment:
Shouldn't be necessary. ItemT and classdesc ensure this happens anyway, automatically for all such ItemT derived types.